appender_test.go

```diff
@@ -0,0 +1,197 @@
+package duckdb
+
+import (
+	"context"
+	"database/sql"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/georgysavva/scany/sqlscan"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	testAppenderTableDdl = `CREATE TABLE appdata(id BIGINT, u8 UTINYINT, i8 TINYINT, u16 USMALLINT, i16 SMALLINT,
+		u32 UINTEGER, i32 INTEGER, u64 UBIGINT, i64 BIGINT,
+		tm TIMESTAMP, f REAL, d DOUBLE, s VARCHAR, b BOOLEAN)`
+)
+
+func openAppenderDB(t *testing.T, pathname string) *sql.DB {
+	db, err := sql.Open("duckdb", pathname)
+	require.NoError(t, err)
+	require.NoError(t, db.Ping())
+	return db
+}
+
+func createAppenderTable(db *sql.DB, t *testing.T) *sql.Result {
+	res, err := db.Exec(testAppenderTableDdl)
+	require.NoError(t, err)
+	return &res
+}
+
+func TestAppenderSimple(t *testing.T) {
```

Comment:
Maybe we could only use TestRandomizedAppender instead of TestAppenderSimple as it already covers everything that the simple test case does.